core/domain/classifier_services.py

```diff
@@ -305,9 +287,9 @@ def _update_classifier_training_job(classifier_training_job_model, status):
     classifier_training_job_model.put()
 
 
-def save_classifier_training_job(algorithm_id, exp_id, exp_version,
-                                 state_name, status, training_data,
-                                 job_id="None"):
+def save_classifier_training_job(algorithm_id, interaction_id, exp_id,
+                                 exp_version, state_name, training_data,
+                                 status, job_id=None):
```

Comment:
@seanlip That was how we were doing it before. You suggested making it like this. If we were passing the domain object, we would have no access to job_id when job doesn't exist. If you remember, I was initially using a dummy job_id approach, which we changed to this way. I think we could make the save_classifier_training_job into create_classifier_training_job and update_classifier_training_job to avoid conflict over the job_id issue. @anmolshkl What do you suggest?